### ISSUE TITLE
Vibration too strong in navigation bar

### ISSUE THREAD

**Issue by desperex** [bdogg718k/Fixed_or_Broken#9]
Even after setting it to min in sound settings it still much stronger comparing to stock htc one

**Comment by Monzerkj** [comment]
yes its very annoying , how did you set it on min I couldn't find option for that ?
